src/pages/changePasswordPage/ChangePasswordPage.jsx

```diff
@@ -0,0 +1,167 @@
+import React, { useState } from "react";
+//External packages here
+import { Popover } from "antd";
+//Supporting utilities files here
+import { validateNewpassword } from "../../utils/validate";
+
+import { checkPasswordCriteria } from "../../components/PasswordRequirement/PasswordRequirement";
+
+//CSS here
+import "./scss/ChangePasswordPage.css";
+const ResetPasswordPage = () => {
+  const [passwordError, setPasswordError] = useState({});
+  const [passwordCriteria, setpasswordCriteria] = useState(null);
+  const [popoverShow, setPopovershow] = useState(false);
+  const [newPassword, setNewPassword] = useState({
+    newPassword: "",
```

Comment:
I have told not to use an object for state on my previous comments

Reply:
I have changed from object to separate state